SU-HKKU/cardinal_service_profile
# CSD-327 Added 1/4 Table WYSWYG Style

# READY FOR REVIEW

# Summary
- Added 1/4 Table WYSWYG Style

# Needed By (Date)
- 08/25

# Urgency
- medium

# Steps to Test

1. Pull this branch
2. Clear site cache
3. Apply `1/4 Table` style to the table on the Glossary page:
http://cardinalservice.ace-gryphon.loc/resources-support/glossary

The first column should now span 25%.

# Associated Issues and/or People
- [CSD-327](https://stanfordits.atlassian.net/browse/CSD-327)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
